File: articles/algorithms-in-media-platforms-and-attention-systems/typescript/attention_risk.ts
const engagementPressure = 0.92;
const creatorImpact = 0.88;
const publicKnowledgeImpact = 0.78;
const userControl = 0.44;
const contestability = 0.42;
const score = (engagementPressure + creatorImpact + publicKnowledgeImpact + (1.0 - userControl) + (1.0 - contestability)) / 5.0;
console.log(`attention_risk_score=${score.toFixed(4)}`);

File: articles/algorithms-in-media-platforms-and-attention-systems/haskell/AttentionRisk.hs
module Main where

main :: IO ()
main = do
  let engagementPressure = 0.92
      creatorImpact = 0.88
      publicKnowledgeImpact = 0.78
      userControl = 0.44
      contestability = 0.42
      score = (engagementPressure + creatorImpact + publicKnowledgeImpact + (1 - userControl) + (1 - contestability)) / 5.0
  putStrLn ("attention_risk_score=" ++ show score)
